locations/hacrafu-fiocchi.yml

```diff
@@ -0,0 +1,72 @@
+---
+
+location: hacrafu-fiocchi
+location_nice: Apothekerteich-Fiocchi-Config
+latitude: 52.52320
+longitude: 13.77234
+contact_nickname: 'HaCraFu'
+contacts:
+  - 'freifunk@hacrafu.de'
+
+hosts:
+
+  - hostname: hacrafu-fiocchi-core
+    role: corerouter
+    model: "dlink_covr-x1860-a1"
+    wireless_profile: freifunk_hacrafu
+
+ipv6_prefix: '2001:bf7:850:b00::/56'
+# dhcp		10.31.204.0/27
+# mesh5		10.31.203.224/32
+# mesh2		10.31.203.225/32
+# mesh-MGMT	10.31.202.95/32
+
+# Disable noping
+dhcp_no_ping: false
+
+networks:
+  # DHCP with filtering and isolation
+  - vid: 40
+    role: dhcp
+    untagged: true
+    inbound_filtering: true
+    enforce_client_isolation: false
+    prefix: 10.31.204.0/27
+    ipv6_subprefix: 0
+    assignments:
+      hacrafu-fiocchi-core: 1
+
+  # MESH - 5 GHz 802.11s
+  - vid: 20
+    role: mesh
+    name: mesh_5g
+    prefix: 10.31.203.224/32
+    ipv6_subprefix: -20
+    mesh_ap: hacrafu-fiocchi-core
+    mesh_radio: 11a_standard
+    mesh_iface: mesh
+
+  # MESH - 2.4 GHz 802.11s
+  - vid: 21
+    role: mesh
+    name: mesh_2g
+    prefix: 10.31.203.225/32
+    ipv6_subprefix: -21
+    mesh_ap: hacrafu-fiocchi-core
+    mesh_radio: 11g_standard
+    mesh_iface: mesh
+
+  # MGMT                      # create a management vlan in which we can reach every device on this site for maintenance
+  - vid: 42
+    role: mgmt
+    prefix: 10.31.202.95/32
+    gateway: 1
+    dns: 1
+    ipv6_subprefix: 1
+    assignments:
+      hacrafu-fiocchi-core: 1
```

Comment:
From looking at the map it looks like you have an directional antenna attached to this device pointing towards ditalini, right? Usually you would use a larger management network here and add directional antennas or switches so the management interface can be accessed.

Reply:
The long link to Ditalini is by Acciedent. It's caused by our configuration process.